_examples/render/presenter/v1/article.go

```diff
@@ -0,0 +1,35 @@
+package v1
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pressly/chi/_examples/render/presenter/v2"
+	"github.com/pressly/chi/render"
+)
+
+// Article presented in API version 1.
+type Article struct {
+	*v2.Article `json:",inline" xml:",inline"`
+
+	Data map[string]bool `json:"data" xml:"data"`
+}
+
+var Presenter = render.NewPresenter(ArticleV2ToV1)
+
+func init() {
+	Presenter.RegisterFrom(v2.Presenter)
+}
+
+func ArticleV2ToV1(ctx context.Context, from *v2.Article) (*Article, error) {
```

Comment:
I'm gonna change this signature to

```go
func(r *http.Request, from *v2.Article) (*Article, error) {}
```

so we can use `r.Host`, `r.URL`, `r.RequestURI`, `r.RemoteAddr`, `r.TSL` etc. in the presenter funcs